setup.py

```diff
@@ -33,7 +33,7 @@ def load_requirements(*requirements_paths):
 
 setup(
     name='edx-submissions',
-    version='1.1.2',
+    version='1.1.3',
```

Comment:
Point of note: [searching for edx-submissions across all edx repos](https://github.com/search?utf8=%E2%9C%93&q=user%3Aedx+edx-submissions&type=Code&ref=searchresults) show that ora2 and edx-platform are the only locations that will need to have their pinned version bumped.